meta/core.md

```diff
@@ -27,6 +27,34 @@ To be able to distinguish Dimensions that are defined inside a Cube from Dimensi
 
 </aside>
 
+### meta:Limit {#Limit}
+
+A type of [annotation](#annotation) which can be used to express a limit or target value for a dimension.
+
+<aside class='example' title='Dimension with a target'>
+
+```turtle
+<cube/shape> sh:property [
+  a cube:KeyDimension ;  
+  sh:path ex:year ;
+] , [
+  a cube:MeasureDimension ;
+  meta:annotation [
+    a meta:Limit ;
+    schema:value 95 ;
+    schema:name "Target 2020" ;
+    meta:annotationContext [
+      sh:path ex:year ;
+      sh:hasValue <https://schema.ld.admin.ch/year/2020> ;
+    ] ;
+  ] ;
+] .
+```
+
+</aside>
+
+`schema:minValue` and `schema:maxValue` can be used to express a range of limit values.
```

Comment:
Good catch. We seem to have dropped that along the way - or at least I think we had that covered once.
In any case, we should include that somehow. 
Either with different classes? I.e. `meta:UpperLimit` and `meta:LowerLimit`? Or maybe more elegant with `schema:minValue`/`schema:maxValue` instead of the generic `schema:value`?

Or would you see a case where a LIMIT could no specified to be an upper or lowr Limit @Rdataflow ? It would not really be a limit without that specification?

Reply:
After a small chat with @Rdataflow, we both agree that the generic `schema:value` is not well suited for `meta:Limit`. At least for what we try to express here. 

One thing we should distinguish is between the mathematical concept of a limit (or limes) and limit as we envisioned here. Here, we think about something like a speed limit. Or some value, which should not be exceeded (or exceedence leads to some consequence), or below which a measurement can't be quantified like a detection limit.

We would argue, that these kinds of limits do generally have an upper or lower meaning associated with them. So the use of `schema:value` can be and should be replaced with `schema:minInclusive` and/or `schema:maxInclusive`, to make the meaning of this limit clearer.

Reply:
@Kronmar-Bafu I do not follow, which part is in your opinion missing from the spec? The snippet shows `schema:limit` but it also says that `minValue/maxValue` can be used.